crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -50,23 +51,30 @@ impl Operations {
         None
     }
 
-    pub fn filter_by_topic(&self, topic_name: &str) -> Vec<(String, Operation)> {
+    pub fn filter_by_topics(
+        &self,
+        topics: Vec<String>,
+        state: &GenericCommandState,
```

Comment:
As per this [comment](https://github.com/thin-edge/thin-edge.io/pull/3189#discussion_r1804756584),  this function doesn't have to do anything with workflow's `GenericCommandState` for the topic. Instead, you need to parse prefix here.

Reply:
resolved by c1311aaf198b4b93fd96ab439ab77f816945e7e3